.gitignore

```diff
@@ -1,4 +1,7 @@
-venv
+venv/
 build
 data
 portfolio.egg-info
+.vscode/
```

Comment:
Better to add `*.pyc` also to this list.

Not needed if we do `export PYTHONDONTWRITEBYTECODE=1` to `~/.bashrc`. But, someone working on this might not and might accidently commit a _.pyc_